### PR TITLE
#3 feat: add scramble button

### PR DESCRIPTION
**Pull request by vlrsmlkv**
Added scramble button.

### CODE REVIEW

File: src/components/App.jsx [hunk 1]
```diff
@@ -17,6 +19,7 @@ const App = () => {
   const ref = useRef(null);
 
   const [isLabelSwitchChecked, setIsLabelSwitchChecked] = useState(false);
+  const [scrambleRotationCounter, setScrambleRotationCounter] = useState(0);
```
Comment (OUghTMS) [inline src/components/App.jsx]: same

File: src/components/App.jsx [hunk 2]
```diff
@@ -27,28 +30,70 @@ const App = () => {
...
+  const onLastRotatingHandler = () => {
+    setIsRotating(false);
+    
+    if (scrambleRotationCounter !== 0) {
+      scrambleRubiksCube();
+      setScrambleRotationCounter(scrambleRotationCounter - 1);
     };
   };
 
+  const getScrambleRandomParams = () => {
```
Comment (OUghTMS) [inline src/components/App.jsx]: move into utils

**Review by OUghTMS — CHANGES_REQUESTED** [review]

File: src/components/App.jsx (hunk 1 above)
Comment (vlrsmlkv) [reply src/components/App.jsx]: done ✅

File: src/components/App.jsx (hunk 2 above)
Comment (vlrsmlkv) [reply src/components/App.jsx]: done ✅

**Review by OUghTMS — APPROVED** [review]

**Pull request merged**